pkg/codegen/testing/test/testdata/entries-function-pp/dotnet/entries-function.cs

```diff
@@ -11,12 +11,12 @@
         3,
     }.Select((v, k) => new { Key = k, Value = v }).Select(entry => 
     {
-        return new Dictionary<string, object?>  
```

Comment:
The code which resolves this type name was broken. Need to revisit the creation of untyped maps in dotnet

Reply:
Related https://github.com/pulumi/pulumi/issues/13623